task-2/Program.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+
+namespace task_2
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine(@"Hi there! I'm the Square calculator
+Enter the side's length of your square in integer:");
+
+            int squareLength = int.Parse(Console.ReadLine());
```

Comment:
Длина стороны может быть произвольным неотрицательным числом, не обязательно целым. Неверный выбор типа данных - это ошибка
```suggestion
Enter the side's length of your square:");

            double squareLength = double.Parse(Console.ReadLine());
```

Имя локальной переменной `squareLength` чересчур вычурное и трудно понимаемое (что такое длина квадрата?). более понятно `sideLength` (длина стороны) или еще проще `side`.